src/Tizen.Multimedia/MediaTool/AudioMediaFormat.cs

```diff
@@ -180,23 +324,38 @@ internal override void AsNativeHandle(IntPtr handle)
         {
             Debug.Assert(Type == MediaFormatType.Audio);
 
-            int ret = Interop.MediaFormat.SetAudioMimeType(handle, MimeType);
+            int ret = Native.SetAudioMimeType(handle, MimeType);
             MultimediaDebug.AssertNoError(ret);
 
-            ret = Interop.MediaFormat.SetAudioChannel(handle, Channel);
+            ret = Native.SetAudioChannel(handle, Channel);
             MultimediaDebug.AssertNoError(ret);
 
-            ret = Interop.MediaFormat.SetAudioSampleRate(handle, SampleRate);
+            ret = Native.SetAudioSampleRate(handle, SampleRate);
             MultimediaDebug.AssertNoError(ret);
 
-            ret = Interop.MediaFormat.SetAudioBit(handle, Bit);
+            ret = Native.SetAudioBit(handle, Bit);
             MultimediaDebug.AssertNoError(ret);
 
-            ret = Interop.MediaFormat.SetAudioAverageBps(handle, BitRate);
+            ret = Native.SetAudioAverageBps(handle, BitRate);
             MultimediaDebug.AssertNoError(ret);
 
-            ret = Interop.MediaFormat.SetAudioAacType(handle, AacType);
+            ret = Native.SetAudioAacType(handle, AacType);
             MultimediaDebug.AssertNoError(ret);
+
+            if (AudioChannelMap != null)
+            {
+                ret = Native.SetAudioChannelMask(handle, GetAudioChannelMask(handle));
+                MultimediaDebug.AssertNoError(ret);
+            }
+        }
+
+        private ulong GetAudioChannelMask(IntPtr handle)
```

Comment:
```suggestion
        private static ulong GetAudioChannelMask(IntPtr handle, IList<MediaFormatAudioChannelPosition> audioChannelMap )
```
How about change to static?

Reply:
OK I'll change it.